core/packages/test/config/launch-config.toml

```diff
@@ -0,0 +1,95 @@
+[settings]
+node_spawn_timeout = 120
+provider = "native"
+timeout = 600
+
+## Rococo
+[relaychain]
+chain = "rococo-local"
+default_command = "/tmp/snowbridge/bin/polkadot"
+default_args = ["-lparachain=debug","--beefy","--enable-offchain-indexing=true","--pruning","archive"]
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+max_validators_per_core = 2
+needed_approvals = 1
+validation_upgrade_cooldown = 2
+validation_upgrade_delay = 2
```

Comment:
Think we should also port over the following from the old config:
```
    "genesis": {
      "runtime": {
        "session_length_in_blocks": 10
      }
    }
```

For testing BEEFY, we may want to alter the session length.

Reply:
already defined in rococo chain spec so should be duplicated
https://github.com/paritytech/polkadot/blob/a779892f8ee2a072ac9676d7949ac4539f630ecd/node/service/src/chain_spec.rs#L1925

Reply:
Can we define `session_length_in_blocks` here and use it in the chain spec?

Reply:
I'm fine with duplicating it in the config. Its there for documentation purposes, so that we know what the current session length is, and how to change it.